sdk/core/azure-core/src/main/java/com/azure/android/core/util/logging/ClientLogger.java

```diff
@@ -0,0 +1,6 @@
+package com.azure.android.core.util.logging;
+
+public class ClientLogger {
```

Comment:
Created stub for my project to compile. This class is a part of PR #129 by @anuchandy.

Reply:
Let's just make sure to remove this before merging.